sdk/keyvault/azsecrets/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.1.2 (Unreleased)
+## 0.2.0 (Unreleased)
```

Comment:
You could sneak the date in here as well

Reply:
I'm going to let the Prepare-Release.ps1 script take care of this